COLLABORATOR_GUIDE.md

```diff
@@ -130,7 +130,7 @@ can be fast-tracked and may be landed after a shorter delay:
 * Changes that fix regressions.
 
 When a pull request is deemed suitable to be fast-tracked, label it with
-`fast-track`. The pull request can be landed once 2 or more collaborators
+`fast-track`. The pull request can be landed once 2 or more Collaborators
```

Comment:
Shouldn't it be `collaborators` instead of `Collaborators ` ? Its not the first word in a new line but part of the line right  ?

Reply:
Collaborator is a title and it should be capitalized.

Reply:
Should we be linting for capitalizing `Collaborator(s)` then?

Reply:
> Shouldn't it be `collaborators` instead of `Collaborators `? 

A case could be made for lower case or capitalizing. While I would prefer lower case myself, I am far more concerned with consistency. Since we capitalize it multiple times elsewhere in this and other documents, capitalizing it here normalizes things without a lot of needless churn.